### ISSUE TITLE
CheckMate-038 data and response label

### ISSUE THREAD

**Issue by hereagain-Y** [sidhomj/DeepTCR_Cancer#3]
Hi, 
Thank you for building up this beautiful tool.
When looked at the paper, I was wondering is there a folder for training data used in this paper, specifically the pre-treat CheckMate-038 cohort data and a response.csv labeled their performance after immunotherapy. I seemed to unable to find it here.
Thank you!
